_posts/2015-02-03-8.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Creating a map"
+date:   2015-02-03 10:00:08
+answerrostag: "simulation_creat_map"
+track: [simulation]
+---
+
+***NOTE:*** *Some parts of this article duplicate information from [TurtleBot Simulation]({{ site.url }}{% post_url 2015-02-01-28 %}) post. We will merge them later.*
```

Comment:
Note, likely that post will become redundant and it will probably be better to have people follow this whole series rather than that one post.